.github/workflows/changelog.yml

```diff
@@ -0,0 +1,25 @@
+name: Changelog
+
+on: [push]
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # https://github.com/actions/checkout/releases/tag/v2.3.4
+
+      - name: Use Node.js 12
+        uses: actions/setup-node@56899e050abffc08c2b3b61f3ec6a79a9dc3223d # https://github.com/actions/setup-node/releases/tag/v1.4.4
+        with:
+          node-version: '12.x'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Assert that a new changelog was added, or that the check is skipped
+        env:
+          TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: node .github/scripts/check-changelog
```

Comment:
I would prefer to have it run in CI-only, it's in the `.github` directory after all. It's a very simple check and documenting it outside would probably bring in more noise that the amount of generated value it would provide (personally, I would probably never, ever run it locally).